bob-common/src/error.rs

```diff
@@ -196,4 +202,5 @@ pub enum Kind {
     RequestFailedCompletely(String),
     DisksEventsLogger(String),
     Unauthorized,
+    HolderTemporaryUnavailable,
```

Comment:
Please, check all places, where Error conversion happened. Example: https://github.com/qoollo/bob/blob/master/bob/src/api/mod.rs#L1129
This new error should be converted properly